syntaxes/z80-macroasm.tmLanguage.json

```diff
@@ -56,7 +56,7 @@
 				},
 				{
 					"name": "keyword.control.z80asm",
-					"match": "(?i:(?<=\\s)(?:align|bank|bankset|limit|protect|write direct|charset|def[irs]|str|struct)\\b)"
+					"match": "(?i:(?<=\\s)(?:align|bank|bankset|limit|protect|write direct|charset|def[irs]|str|struct|endstruct)\\b)"
```

Comment:
- `align` and `charset` already defined few lines above -> **remove**
- `def[irs]` merge character enumerations with already defined few lines above -> **`def[bdlmswir]`**
- whitespace between `write` and `direct` should be matched nicely and "less-whitespace-strict" with `\\s+`